### ISSUE TITLE
[Bug Report] Addon action blocked

### ISSUE THREAD

**Issue by fuda01** [Kiatra/BlizzMove#133]
### What game version did the bug occur in?

Retail

### Troubleshooting

- [X] I have verified that BlizzMove is up to date.
- [X] I have tried to reproduce the bug with no other addons loaded.
- [X] I have verified that the issue hasn't yet been reported.

### Describe the bug.

LUA error occurs random and I can't use any spells anymore. Everything works again after a reload. 

### Steps to reproduce.

I don't know how to reproduce that problem. It happens very random.

### Expected behavior.

Not give me the lua error and not lock me out of my spells?

### Errors

```lua
3x [ADDON_ACTION_BLOCKED] AddOn 'BlizzMove' tried to call the protected function 'Frame:SetPassThroughButtons()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `SetPassThroughButtons'
[string "@Blizzard_MapCanvas/MapCanvas_DataProviderBase.lua"]:224: in function `CheckMouseButtonPassthrough'
[string "@Blizzard_MapCanvas/Blizzard_MapCanvas.lua"]:209: in function `AcquirePin'
[string "@WorldQuestTracker/WorldQuestTracker_Core.lua"]:1455: in function `ShowLine'
[string "@WorldQuestTracker/WorldQuestTracker_Core.lua"]:1569: in function <WorldQuestTracker/WorldQuestTracker_Core.lua:1550>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
}

```

**Comment by fuda01** [comment]
More errors:

`1x [ADDON_ACTION_BLOCKED] AddOn 'BlizzMove' tried to call the protected function 'MultiBar7Button9:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `SetAttribute'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:488: in function `UpdatePressAndHoldAction'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:554: in function `Update'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:506: in function `UpdateAction'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:991: in function `OnEvent'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:207: in function <Blizzard_ActionBar/Mainline/ActionButton.lua:204>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
}
`

`1x [ADDON_ACTION_BLOCKED] AddOn 'BlizzMove' tried to call the protected function 'MultiBar5Button1:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `SetAttribute'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:488: in function `UpdatePressAndHoldAction'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:554: in function `Update'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:506: in function `UpdateAction'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:991: in function `OnEvent'
[string "@Blizzard_ActionBar/Mainline/ActionButton.lua"]:207: in function <Blizzard_ActionBar/Mainline/ActionButton.lua:204>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
}
`

**Comment by Numynum** [comment]
the first error is unrelated, not caused by blizzmove, and mostly harmless (it's a known issue for effectively all addons that add icons to the map, up to blizzard to fix)

the other 2 issues are unlikely to be caused by blizzmove, but more likely that some other addon uses the same libraries as blizzmove, which causes us to get blamed
most/all bag addons are currently known to have this issue for some players; and some other different addons as well
good news, is that using an actionbar addon like bartender or elvui makes you immune to this issue; and blizzard shared there is a fix planned for 11.0.5, which should avoid this issue as well
